home/src/main/resources/rdf/i18n/sr_Latn_RS/interface-i18n/firsttime/vitro_UiLabel.ttl

```diff
@@ -743,6 +745,7 @@ uil-data:containers_do_not_pick_up_changes.Vitro
 uil-data:file_upload_error_media_type_not_allowed.Vitro
         rdf:type         owl:NamedIndividual ;
         rdf:type         uil:UILabel ;
+        rdfs:label       "Fajl media tip nije dozvoljen. Media tip otpremljenog fajla je {0}."@sr-Latn-RS ;
```

Comment:
Maybe we should change "fajl media tip" into "media tip fajla", it sounds more natural... for me at least